appeals/web/src/server/appeals/appeal-details/lpa-questionnaire/lpa-questionnaire.mapper.js

```diff
@@ -229,6 +237,7 @@ export async function lpaQuestionnairePage(
 			caseSummary,
 			...appealTypeSpecificPageComponents,
 			additionalDocumentsSummary,
+			insetTextComponent,
```

Comment:
Will this appear above the radios? I think the instructions say above the button.

Reply:
Good spot - I didn't think about this for some reason! Getting the inset text to appear right above the button was trickier than it seemed, I had to modify the display page pattern template. The template renders "form" PageComponents (i.e. radios and checkboxes) separately within a form tag at the bottom of the page (as this is the pattern for display pages), which meant the inset text was always rendering above the radios regardless of its place in the pageComponents array. Now, any inset-text PageComponent which is after any "form" PageComponent in the array will be rendered alongside the "form" components.